migrations/00000000000000-init.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
       .collection("jobs")
       .createIndexes([
         { key: { id: 1 } },
-        { key: { job_type: 1, internal_id: -1 } },
+        { key: { job_type: 1, internal_id: -1 }, unique: true },
         { key: { job_type: 1, status: 1, internal_id: -1 } },
```

Comment:
Does it make sense to not have `unique` on `{ key: { job_type: 1, status: 1, internal_id: -1 } },`
Could you give an example?

Reply:
I don't think so. If `job_type` and `internal_id` are already unique then `job_type`, `internal_id` and `status` will also always be unique. I don't see an advantaged in asking MongoDB to check for that as well (or maybe MongoDB optimises and skips it itself). Am I missing something?